research/configs/fractal_config.py

```diff
@@ -16,10 +16,78 @@ def mlp1h():
     )
 N_FREE_PARAMS_1 = N_FRAC_1 * N_FRAC_1 + N_FRAC_1 * 1
 
-def mean_field_parametrization():
+def parametrization(type, opt='adam', alignment='full', n_layers=3):
     from parametrization import abc_parametrization
-    # NOTE: overfit to mlp1h model config
-    al, bl, cl = [0.0, 1.0], [0.0, 0.0], [-1.0, -1.0] # mean field
+
+    if type == 'mup':
```

Comment:
I think if you're writing code like this it makes more sense to just make a parametrization config for each. The naming for this function is a bit ambiguous. f.e. is a0 = -100, b0 = -1000 not a parametrization?

I'd suggest to make a config for each of them. Each can have optimizer and alignment args since that semantically makes sense to me - f.e. mean_field_parametrization can exist under different optimizer and alignment assumptions.

Reply:
Ok, no problem